corehq/apps/reports/static/reports/js/report_filter.drilldown_options.js

```diff
@@ -1,82 +1,82 @@
-var DrilldownOptionFilterControl = function (options) {
-    var self = this;
+hqDefine('reports/js/report_filter/drilldown_options', function() {
+    var DrilldownOptionFilterControl = function (options) {
+        var self = {};
 
-    self.notification = new DrilldownFinalNotification(options.notifications);
-    self.controls = ko.observableArray(ko.utils.arrayMap(options.controls, function (select) {
-        return new DrilldownOption(select, options.drilldown_map);
-    }));
+        self.notification = DrilldownFinalNotification(options.notifications);
```

Comment:
When changing from classical to functional inheritance, the naming convention switches from ucfirst to lcfirst - https://github.com/dimagi/commcare-hq/pull/19895#pullrequestreview-106915955

Reply:
ooh nice, hadn't seen that convo ^^